src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1581,9 +1582,14 @@ ssize_t sockinfo_udp::tx(const tx_call_t call_type, const struct iovec* p_iov, c
 			si_udp_logdbg("MSG_OOB not supported in UDP (tx-ing to os)");
 			goto tx_packet_to_os;
 		}
-
+		
 		sock_addr dst((struct sockaddr*)__dst);
-
+	
+		if (TRANS_VMA != find_target_family(ROLE_UDP_SENDER, dst.get_p_sa(), m_bound.get_p_sa())) {
+			setPassthrough();
+			goto tx_packet_to_os;
+		}	
+		
```

Comment:
If there is a configuration rule on DNS port - why doing setPassthrough() to the socket? 
Can't it be there is other offloaded traffic on this socket?